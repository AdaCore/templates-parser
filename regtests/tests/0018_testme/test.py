from test_support import *

run('testme', ["testme31.tmplt"])
run('print_tree', ["testme31.tmplt"])
