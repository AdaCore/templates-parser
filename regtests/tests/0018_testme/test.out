


Header

Hello


Footer

base!

[INFO] testme31.tmplt
 -> dir/file.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[INCLUDE] dir/file.tmplt
    $0 = dir/file.tmplt
  [INFO] dir/file.tmplt
 -> dir/subdir/footer.tmplt
 -> dir/header.tmplt
  [C_INFO]  1 FALSE
  [TEXT] 
  [INCLUDE] dir/header.tmplt
      $0 = header.tmplt
    [INFO] dir/header.tmplt
    [C_INFO]  1 FALSE
    [TEXT] 
    [TEXT] Header
  [TEXT] 
  [TEXT] Hello
  [TEXT] 
  [INCLUDE] dir/subdir/footer.tmplt
      $0 = subdir/footer.tmplt
    [INFO] dir/subdir/footer.tmplt
 -> base.tmplt
    [C_INFO]  1 FALSE
    [TEXT] 
    [TEXT] Footer
    [TEXT] 
    [INCLUDE] base.tmplt
        $0 = /base.tmplt
      [INFO] base.tmplt
      [C_INFO]  1 FALSE
      [TEXT] base!

